code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -894,15 +880,6 @@
 	recoil = 25
 	added_damage_bullet_pve = 15
 
-/obj/item/projectile/bullet/shotgun/payload/on_impact(atom/target)
```

Comment:
why are you removing the payload shots bomb?

Reply:
shit, you're right - I noticed I'd done that, but thought I'd un-removed that before the PR. my mistake, let me just fix that